test/defs/test_defs_de.rb

```diff
@@ -23,11 +23,11 @@ def test_de
   assert_equal name, (Holidays.on(date, :de, :informal)[0] || {})[:name], "Failed on '#{name}' for date '#{date}'"
 end
 
-[:de_bw, :de_by, :de_st, :de_].each do |r|
+[:de_bw, :de_by, :de_by_aux, :de_st, :de_].each do |r|
```

Comment:
Was this added because of the changes in https://github.com/holidays/definitions/pull/16? If so, you can remove them from this PR, I will handle re-generating the files for you. You don't have to worry about it! I would prefer if the PR just concentrated on the README and the actual bug in question, I'll handle the definitions. 👍